modules/uhttp/http_test.go

```diff
@@ -42,7 +42,7 @@ import (
 )
 
 // Custom default client since http's defaultClient does not set timeout
-var _defaultClient = &http.Client{Timeout: 5 * time.Second}
+var _defaultHTTPClient = &http.Client{Timeout: 5 * time.Second}
```

Comment:
hmm, TODO for future, this 5 second timeout seems like it should be configurable. maybe 2 seconds to start out with?